.github/CODEOWNERS

```diff
@@ -118,6 +118,12 @@
 /sdk/cognitiveservices/cognitiveservices-qnamaker-runtime/ @bingisbestest @nerajput1607
 /sdk/cognitiveservices/cognitiveservices-qnamaker/ @bingisbestest @nerajput1607
 
+# PRLabel: %MixedReality
```

Comment:
The label to use is `Mixed Reality`. See https://github.com/Azure/azure-sdk-for-js/labels?q=mixed

```suggestion
# PRLabel: %Mixed Reality
```